plugins/linters/reek/plugin.toml

```diff
@@ -0,0 +1,20 @@
+config_version = "0"
+
+[plugins.definitions.reek]
+runtime = "ruby"
+package = "reek"
+file_types = ["ruby"]
+latest_version = "6.3.0"
+known_good_version = "6.3.0"
+version_command = "reek --version"
+config_files = [".reek.yml"]
+description = "Ruby linter"
+
+[plugins.definitions.reek.drivers.lint]
+script = "reek --format json ${target}"
+success_codes = [0, 2]
```

Comment:
Is there documentation for these 0 and 2 exit codes being success?

I am not 100% sure but I am wondering if it would be a good idea to comment the plugin definition code with references to the relevant docs to make it easier on someone making changes later not needing to start from scratch.  

Thoughts?

Reply:
Unfortunately, the only reference to these exit codes is if you run `reek --help` which informs you what the default exit codes are and how to override them if you'd like.

Reply:
OK, we could capture that information into a comment for the benefit of a future developer

While I enjoy having short and sweet plugin definitions, I don't see a harm in commenting them